tailwind.config.mjs

```diff
@@ -41,6 +43,7 @@ export default {
           inverted: '#f2f2f2',
         },
         blue: {
+          focus: '#59a7e8',
```

Comment:
Would it make sense to give this `#59a7e8` color a name as it is repeated in a few places? Since this is a JS file you could just use a variable + string interpolation for it.

Reply:
Yup, I had considered holding off while this local config was in flux (presumably we'll move focus-blue to the upstream Tailwind preset in `frontend-shared` as part of the next "Phase"), but I can do it now.